### PR TITLE
Update requirements to Python 3.6 and Django 1.11

### PR DESCRIPTION
**Pull request by tuchfarber**
This updates requirements to Python 3.6 and Django 1.11.10. Mostly a simple upgrade, except that in Django 1.9, compile translations traverses your whole directory tree and Sphinx 1.7.0+ has a broken translation. So I had to pin 1.6.7 until they accept my PR or fix the translation issue.

### CODE REVIEW

**Comment by tuchfarber** [comment]
Hey! Tests are passing. Good for a review whenever you have time Jeremy.

File: {{cookiecutter.repo_name}}/requirements/docs.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Sphinx
+Sphinx==1.6.7 ## Pinned due to bug in 1.7.0 - 1.7.1
```
Comment (jmbowman) [inline {{cookiecutter.repo_name}}/requirements/docs.txt]: Which bug?  If there's a GitHub issue for it, please add a link so we know when to unpin it again

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/apps/core/models.py [hunk 2]
```diff
@@ -26,5 +26,5 @@ class Meta(object):  # pylint:disable=missing-docstring
     def get_full_name(self):
         return self.full_name or super(User, self).get_full_name()
 
-    def __unicode__(self):
-        return unicode(self.get_full_name())
+    def __str__(self):
```
Comment (jmbowman) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/apps/core/models.py]: This should use the [python_2_unicode_compatible](https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.encoding.python_2_unicode_compatible) decorator so it works cleanly under all supported Python versions

**Review by jmbowman — APPROVED** [review]
Looks good other than the 2 noted nitpicks.

**Pull request merged**